bsg_fpu/README.md

```diff
@@ -1,97 +1,18 @@
-# bsg_fpu
```

Comment:
Why remove these comments? These seem relevant and bring in a bunch of details that are difficult to infer from the code.

Reply:
revised the README.